com.cubrid.cubridmigration.core/src/com/cubrid/cubridmigration/core/engine/event/MigrationXLSNoSupportEvent.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2009 Search Solution Corporation. All rights reserved by Search Solution.
```

Comment:
Copyright needs to be modified to '2016 CUBRID Corporation'

Reply:
Thanks for the advice, fixed it.